agent/handlers/v3/container_metadata_handler.go

```diff
@@ -81,7 +81,7 @@ func GetContainerResponse(containerID string, state dockerstate.TaskEngineState)
 func GetContainerNetworkMetadata(containerID string, state dockerstate.TaskEngineState) ([]containermetadata.Network, error) {
 	dockerContainer, ok := state.ContainerByID(containerID)
 	if !ok {
-		return nil, errors.Errorf("Unable to find container '%s'", containerID)
+		return []containermetadata.Network{}, nil
```

Comment:
could you add a comment here mentioning that for pulled containers, network metadata may not be present yet so we are not returning an error? same below.

Reply:
were we able to test this for v2 metadata?

Reply:
The comment is added in [this commit](https://github.com/aws/amazon-ecs-agent/pull/2700/commits/fba2d68cb35ed883fb1b920417c02caa77a43981#diff-7bc0cfcaf091a1a1d35c0a7ee5aa7e7aa7ba93c6b298e19057c74a27d32b12beR83) for v3 and [here](https://github.com/aws/amazon-ecs-agent/pull/2700/commits/fba2d68cb35ed883fb1b920417c02caa77a43981#diff-b5891838cbae3e4a2488417b501f69713e0f292a6c106b1fa17357b32e196070R84) fo v4. Thank you!